ome_zarr/scale.py

```diff
@@ -123,19 +127,21 @@ def __create_group(
 
     def nearest(self, base: np.ndarray) -> List[np.ndarray]:
         """
-        Downsample using :func:`cv2.resize`.
+        Downsample using :func:`skimage.transform.resize`.
 
         The :const:`cvs2.INTER_NEAREST` interpolation method is used.
         """
         return self._by_plane(base, self.__nearest)
 
     def __nearest(self, plane: np.ndarray, sizeY: int, sizeX: int) -> np.ndarray:
         """Apply the 2-dimensional transformation."""
-        return cv2.resize(
+        return resize(
             plane,
-            dsize=(sizeY // self.downscale, sizeX // self.downscale),
-            interpolation=cv2.INTER_NEAREST,
-        )
+            output_shape=(sizeY // self.downscale, sizeX // self.downscale),
```

Comment:
@will-moore : can you check that this matches your cv2 use cases?

Reply:
I think the output_shape needs to be `(sizeX, sizeY)` as in https://github.com/ome/ome-zarr-py/pull/71/files#diff-47d7162e6c0191b1b2d4a31da6e1445085655c7a1e63354b488b1d53ec54d1a2R421
See examples at https://www.tutorialkart.com/opencv/python/opencv-python-resize-image/ with `dim = (width, height)`

Reply:
Ok, I will double check this and see if we can cover it by some test.

Reply:
I have added tests for the scaler in e07194d. Judging from the results the current implementation is correct.